### PR TITLE
CMS Kit - Tag Refactoring

### PR DESCRIPTION
**Pull request by ilkayilknur**
- Some missing attributes have been added. 
- I've made some refactoring and changes on tag methods.

### CODE REVIEW

File: modules/cms-kit/src/Volo.CmsKit.Public.Application.Contracts/Volo/CmsKit/Public/Tags/ITagAppService.cs [hunk 1]
```diff
@@ -7,6 +7,6 @@ namespace Volo.CmsKit.Public.Tags
 {
     public interface ITagAppService : IApplicationService
     {
-        Task<List<TagDto>> GetAllRelatedTagsAsync(GetRelatedTagsInput input);
+        Task<List<TagDto>> GetAllRelatedTagsAsync(string entityType, string entityId);
```
Comment (enisn) [inline modules/cms-kit/src/Volo.CmsKit.Public.Application.Contracts/Volo/CmsKit/Public/Tags/ITagAppService.cs:10]: I'm not sure about this changes, as @cotur 's say each input and output parameters should be object in an AppService. That's why we created 'GetRelatedTagsInput' type.

**Review by enisn — APPROVED** [review]
Rest of them LGTM

File: modules/cms-kit/src/Volo.CmsKit.Public.Application.Contracts/Volo/CmsKit/Public/Tags/ITagAppService.cs (hunk 1 above)
Comment (ilkayilknur) [reply modules/cms-kit/src/Volo.CmsKit.Public.Application.Contracts/Volo/CmsKit/Public/Tags/ITagAppService.cs:10]: All other application services and controllers use the same convention. That's why I've made the changes. If those implementations are against the convention we can change the other usages.

Comment (enisn) [reply modules/cms-kit/src/Volo.CmsKit.Public.Application.Contracts/Volo/CmsKit/Public/Tags/ITagAppService.cs:10]: Ok, it's better now. 👍

**Review by cotur — APPROVED** [review]

**Pull request merged**